test/test.js

```diff
@@ -0,0 +1,10 @@
+const request = require('supertest');
+const app = require('../app');
+
+describe('GET v1/status', function() {
```

Comment:
This is an integration test. Would place them under `/test/integration/v1/status-test.js`